src/sql/workbench/browser/designer/interfaces.ts

```diff
@@ -187,9 +187,9 @@ export interface DesignerTableComponentRowData {
 
 
 export enum DesignerEditType {
-	Add = 0,
```

Comment:
have you tested this change? this will break the STS, I believe STS is expecting numbers.

Reply:
please address this comment